ORST/Assets/Scripts/Core/DialoguesTutorial/Scriptable/TutorialDialogueNode.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Sirenix.OdinInspector;
+using System;
+using UnityEngine;
+
+namespace ORST.Core.Dialogues {
+    [Serializable]
+    public class TutorialDialogueNode {
+        [SerializeField] private string m_Title;
+        [SerializeField, TextArea(5,15)] private string m_Text;
+        [SerializeField] private Sprite[] m_Images;
+        [SerializeField] private bool m_ShowPopup;
+        [SerializeField, ShowIf("m_ShowPopup")] private string m_PopupTitle;
+        [SerializeField, ShowIf("m_ShowPopup"), TextArea(3, 10)] private string m_PopupText;
+        [SerializeField, ShowIf("m_ShowPopup")] private bool m_SnapPopupToPosition;
+        [SerializeField, ShowIf("m_SnapPopupToPosition")] private Transform m_SnapPosition;
```

Comment:
Let's use `nameof` so that if we rename fields these will be renamed automatically.

```suggestion
        [SerializeField, ShowIf(nameof(m_ShowPopup))] private string m_PopupTitle;
        [SerializeField, ShowIf(nameof(m_ShowPopup)), TextArea(3, 10)] private string m_PopupText;
        [SerializeField, ShowIf(nameof(m_ShowPopup))] private bool m_SnapPopupToPosition;
        [SerializeField, ShowIf(nameof(m_SnapPopupToPosition))] private Transform m_SnapPosition;
```